src/pages/contribute/components/Tooltip.js

```diff
@@ -1,8 +1,7 @@
 import React from 'react';
-import './ToolTip.sass';
-import './DonateModal.sass';
+import './Tooltip.sass';
```

Comment:
```suggestion
import './ToolTip.sass';
```

You need to change this or rename the file.

Reply:
Hm there was a bug with git and renaming the file. I fixed it, thank you!